src/nIsForName.js

```diff
@@ -21,8 +21,13 @@
  *
  * @return {string[]}
  */
+function checkN(element) {
+  var words = element.split(' ');
```

Comment:
Do not use var, never, [prefer-const](https://mate-academy.github.io/style-guides/javascript.html#references--prefer-const)